cytoscape-matterjs.js

```diff
@@ -0,0 +1,428 @@
+;(function(){ 'use strict';
+
+  // registers the extension on a cytoscape lib ref
+  var register = function( cytoscape ){
```

Comment:
`register = function( cytoscape, Matter ){ ...` to take in dependency (may not be global)